### ISSUE TITLE
skyfield database download breaks sometimes

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#431]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/431) by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

sometimes the url [http://maia.usno.navy.mil/ser7/deltat.data](http://maia.usno.navy.mil/ser7/deltat.data) shows a HTTP Error 503: Service Unavailable

‌

We should find a way to have that data mirrored and tested on one of the servers available without a crash of the testing routine

```
CRITICAL root:mss_qt.py:172 MSS Version: 1.8.1
CRITICAL root:mss_qt.py:173 Python Version: 3.7.3 (default, Mar 27 2019, 22:11:17) 
                            [GCC 7.3.0]
CRITICAL root:mss_qt.py:174 Platform: Linux-4.15.0-54-generic-x86_64-with-debian-buster-sid ((&#39;64bit&#39;, &#39;&#39;))
CRITICAL root:mss_qt.py:175 Fatal error: Traceback (most recent call last):
                              File &#34;/home/user/miniconda3/envs/mssdev/lib/python3.7/site-packages/skyfield/iokit.py&#34;, line 477, in download
                                connection = urlopen(url)
                              File &#34;/home/user/miniconda3/envs/mssdev/lib/python3.7/urllib/request.py&#34;, line 222, in urlopen
                                return opener.open(url, data, timeout)
                              File &#34;/home/user/miniconda3/envs/mssdev/lib/python3.7/urllib/request.py&#34;, line 531, in open
                                response = meth(req, response)
                              File &#34;/home/user/miniconda3/envs/mssdev/lib/python3.7/urllib/request.py&#34;, line 641, in http_response
                                &#39;http&#39;, request, response, code, msg, hdrs)
                              File &#34;/home/user/miniconda3/envs/mssdev/lib/python3.7/urllib/request.py&#34;, line 569, in error
                                return self._call_chain(*args)
                              File &#34;/home/user/miniconda3/envs/mssdev/lib/python3.7/urllib/request.py&#34;, line 503, in _call_chain
                                result = func(*args)
                              File &#34;/home/user/miniconda3/envs/mssdev/lib/python3.7/urllib/request.py&#34;, line 649, in http_error_default
                                raise HTTPError(req.full_url, code, msg, hdrs, fp)
                            urllib.error.HTTPError: HTTP Error 503: Service Unavailable

                            During handling of the above exception, another exception occurred:

                            Traceback (most recent call last):
                              File &#34;/home/user/PycharmProjects/mss/mslib/msui/topview.py&#34;, line 243, in openTool
                                widget = rs.RemoteSensingControlWidget(view=self.mpl.canvas)
                              File &#34;/home/user/PycharmProjects/mss/mslib/msui/remotesensing_dockwidget.py&#34;, line 60, in __init__
                                self.timescale = self.load.timescale()
                              File &#34;/home/user/miniconda3/envs/mssdev/lib/python3.7/site-packages/skyfield/iokit.py&#34;, line 275, in timescale
                                data = self(&#39;deltat.data&#39;)
                              File &#34;/home/user/miniconda3/envs/mssdev/lib/python3.7/site-packages/skyfield/iokit.py&#34;, line 185, in __call__
                                download(url, path, self.verbose)
                              File &#34;/home/user/miniconda3/envs/mssdev/lib/python3.7/site-packages/skyfield/iokit.py&#34;, line 479, in download
                                raise IOError(&#39;cannot get {0} because {1}&#39;.format(url, e))
                            OSError: cannot get http://maia.usno.navy.mil/ser7/deltat.data because HTTP Error 503: Service Unavailable
```

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

this is similiar to #467

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Issue #467 was marked as a duplicate of this issue.

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

@{joernu76} is that an option?  
[https://docs.astropy.org/en/stable/api/astropy.coordinates.get\_body.html](https://docs.astropy.org/en/stable/api/astropy.coordinates.get_body.html)

[https://docs.astropy.org/en/stable/api/astropy.coordinates.get\_sun.html](https://docs.astropy.org/en/stable/api/astropy.coordinates.get_sun.html)

‌

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

use shipped files from skyfield package, fixes #431

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Merged in ReimarBauer/mss/i431 (pull request #723)

use shipped files from skyfield package, fixes #431

Approved-by: Joern Ungermann <j.ungermann@fz-juelich.de>
